GOLD-171 : Confirm ntp offset timestamp is within expected range

abdulazeem-tk4vr opened the pull request:
Linear Task : https://linear.app/shm/issue/GOLD-171/confirm-ntp-offset-timestamp-is-within-expected-range-not-massively

ahmxdiqbal committed:
adjust logFlag and interval for memory logging

adjust interval for memory logging to 300s

abdulazeem-tk4vr committed:
chore: Update time synchronization logic

chore: Add debug flag for NTP bogus offset decrements


chore: Update time synchronization error messages and comment debug endpoint

chore: Remove commented debug endpoint for NTP bogus offset decrements

S0naliThakur committed:
error handling for all external endpoints

update endpoint-exception counter message

kgmyatthu commented on src/state-manager/index.ts:
```diff
@@ -2294,6 +2294,11 @@ class StateManager {
       res.write(response)
       res.end()
     })
+    // Context.network.registerExternalGet('debug-ntp-bad-offset', isDebugModeMiddleware, (_req, res) => {
```
Need revision on why this code is commented out.

kgmyatthu reviewed (dismissed):
LGTM

arhamj committed:
feat(counters): added gossip type to gossip handler exception counter

S0naliThakur committed:
add catch for apoptosize endpoint

code clean up

S0naliThakur committed:
remove unused non binary internal endpoints

block non binary internal point registration

 remove request_receipt_for_tx_old

unblock non binary internal point registration
